net/disk_cache/cobalt/cobalt_backend_impl.cc

```diff
@@ -123,28 +86,20 @@ CobaltBackendImpl::~CobaltBackendImpl() {
   simple_backend_map_.clear();
 }
 
-void CobaltBackendImpl::UpdateSizes(ResourceType type, uint32_t bytes) {
```

Comment:
Return value changed to bool so that the caller can check for early return from the line below and not touch persistent settings if not valid.

Reply:
Thanks for adding these comments. They make code review much easier~